README.md

```diff
@@ -43,6 +43,8 @@ spec:
   noCache: false
   version: v0.3.0
   enableAI: true
+  filters:
```

Comment:
Can we mention this in the readme but not put this as the default? 
It's not going to be a great "wow" moment when people don't see any results because its filtering on Ingress the first time

Reply:
We could leave it displayed in the yaml but comment it by default? What do you think?